sklearn/metrics/_classification.py

```diff
@@ -1254,7 +1254,7 @@ def f1_score(
     average : {'micro', 'macro', 'samples', 'weighted', 'binary'} or None, \
             default='binary'
         This parameter is required for multiclass/multilabel targets.
-        If ``None``, the scores for each class are returned. Otherwise, this
```

Comment:
Currently we use 'score' here but revert to metrics (see below). I am actually unsure about whether to change all 'metrics' to 'scores' for `f1_score`, `precision_score` and `recall_score`, or leave as 'metrics'?

Reply:
good question

Reply:
My own definition would be the following:

- "score": scalar where higher value is better
- "error": scalar where lower value is better
- "metric": term accounting for "score" and "error" to measure the statistical performance.

Therefore, I would be fine to generalize the term "score" when it comes these functions.

Reply:
Sorry, do you mean 'generalize "metric"' ?

Reply:
Or more specifically, do you mean use 'metric' everywhere as it accounts for both types?